src/index.js

```diff
@@ -48,66 +48,67 @@ const FIELDS = [
   'urls'
 ]
 
-function initCozyClient() {
-  const cozyCredentials = JSON.parse(process.env.COZY_CREDENTIALS)
-  return new CozyClient({
-    uri: process.env.COZY_URL,
-    token: cozyCredentials.token.accessToken
-  })
-}
-
-async function getAllContacts(client) {
-  let allContacts = []
-  const DOCTYPE_CONTACTS = 'io.cozy.contacts'
-  const contactsCollection = client.collection(DOCTYPE_CONTACTS)
-  let hasMore = true
-  while (hasMore) {
-    const resp = await contactsCollection.all()
-    allContacts = [...allContacts, ...resp.data]
-    hasMore = resp.next
-  }
-
-  return allContacts
-}
-
 /**
  * @param  {} fields:
  * @param {} fields.access_token: a google access token
  * @param {} fields.refresh_token: a google refresh token
  */
 async function start(fields, doRetry = true) {
   log('info', 'Starting the google connector')
+  // const accountID = JSON.parse(process.env.COZY_FIELDS).account
+  const accountID = 'fakeAccountId'
```

Comment:
This would need to change before merging, I think we can use the env variable all the time no?

Reply:
`COZY_FIELDS` is `undefined`. Maybe I should just inject a value in it during dev and add something about it in the README?

Reply:
Yep, sounds good to me. Or something like `const accountID = process.env.NODE_ENV === 'production' ? process.env.COZY_FIELDS.account : 'fakeAccountId'` would also work for us I think.